gensim/summarization/keywords.py

```diff
@@ -10,7 +10,12 @@
 from gensim.summarization.commons import remove_unreachable_nodes as _remove_unreachable_nodes
 from gensim.utils import to_unicode
 from itertools import combinations as _combinations
-from six.moves.queue import Queue as _Queue
+import sys
+is_py2 = sys.version[0] == '2'
+if is_py2:
+    import Queue as queue
```

Comment:
Hi tmylk, that's actually on purpose. There's a line below that says "queue=_Queue", but that's the only place where _Queue is used so i figured I would just delete that line and use queue, I can change it back to _Queue and resubmit this pull request if that would be better received